### PR TITLE
homework 1 js

### PR DESCRIPTION
**Pull request by MashaLoban**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+test += '1';
+console.log(test); //21
+test -= 1;
+console.log(test); //20
+test = Boolean(test); //true
+console.log(test);
+
+//Task 3
+let arr = [2, 3, 5, 8];
+let newArr = 1;
```
Comment (BubentsovKirill) [inline main.js:22]: That doesn't arr, change the name like `result`.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+test += 1; 
+test += '1';
+console.log(test); //21
+test -= 1;
+console.log(test); //20
+test = Boolean(test); //true
+console.log(test);
+
+//Task 3
+let arr = [2, 3, 5, 8];
```
Comment (BubentsovKirill) [inline main.js:21]: Should be `const`.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+let newArr = 1;
+for (let i = 0; i < arr.length; i++) {
+    newArr = newArr * arr[i];
+}
+console.log('Ответ Task3:', newArr);
+
+//Task4
+let arr2 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (BubentsovKirill) [inline main.js:29]: Should be const.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+for (let i = 0; i < arr2.length; i++) {
+    if (arr2[i] > 5 && arr2[i] < 10) {
+        // debugger;
+        console.log('Ответ Task4:', arr2[i]);
+    }
+}
+
+//Task 5
+let arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (BubentsovKirill) [inline main.js:38]: Should be const.

File: main.js (hunk 1 above)
Comment (MashaLoban) [reply main.js:22]: ok, boss

File: main.js (hunk 2 above)
Comment (MashaLoban) [reply main.js:21]: fixed

File: main.js (hunk 3 above)
Comment (MashaLoban) [reply main.js:29]: fixed

File: main.js (hunk 4 above)
Comment (MashaLoban) [reply main.js:38]: done

**Pull request merged**